sbnana/SBNAna/NuMINumuXSec/ICARUSNumuXsec_Cuts.h

```diff
@@ -0,0 +1,112 @@
+#ifndef ICARUSNumuXsec_Cuts_h
```

Comment:
We normally use `#pragma once` in CAFAna/SBNAna code, rather than full include guards.